.gitignore

```diff
@@ -1,5 +1,5 @@
+*GENERATED*
 /output/
 /vendor/
-/fixtures-local/
```

Comment:
*GENERATED* is a replacement for output, not for fixtures-local

Reply:
Done